rpm_spec/manageiq.spec.in

```diff
@@ -126,6 +126,8 @@ popd
 #copy all files/directories below COPY
 %{__cp} -r ./%{appliance_builddir}/COPY/* %{buildroot}/
 
+# symlink webpack manifest
+ln -s %{app_root}/public/packs/webpack-modules-manifest.json %{buildroot}%{manifest_root}/webpack-modules-manifest.json
```

Comment:
We don't actually need the file in the packs directory (in fact it's probably better _not_ in that directory, so it's not accessible via API call), then we can `mv` it.

Reply:
Do we need the file renamed to have underscores for consistency?

Reply:
Moved the file and changed to use underscore.